fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/keyspace/ExtendedDirectoryLayerTest.java

```diff
@@ -99,12 +99,12 @@ private static void testReadCompatible(LocatableResolver writer, LocatableResolv
         Map<String, Long> allocations = new HashMap<>();
         for (int i = 0; i < 50; i++) {
             String key = "key-" + i;
-            Long value = writer.resolve(null, key).join();
+            Long value = writer.resolve((FDBStoreTimer)null, key).join();
```

Comment:
Perhaps these tests could be improved by yet another overload that _only_ took the key and did the `null` itself.

Reply:
Yeah, though it's a little weird that the optional parameter is the first (rather than the last).

Reply:
I added this as a separate commit with this PR, for size